binding/python/rhino.py

```diff
@@ -133,17 +139,16 @@ def delete(self):
     def process(self, pcm):
         """
         Processes a frame of audio and emits a flag indicating if the inference is finalized. When finalized,
-        '.is_understood()' should be called to check if the spoken command is considered valid.
+        `.is_understood()` should be called to check if the spoken command is considered valid.
```

Comment:
No longer necessary since this is returned with get_inference. get_inference can be called which includes whether the command was understood (in-context) or not.